spec/models/aggregations/search_last_three_months_spec.rb

```diff
@@ -30,26 +30,47 @@
     expect(subject.pluck(:dimensions_edition_id)).to match_array([edition1.id, edition2.id])
   end
 
-  describe 'Boundaries' do
-    it 'does not include metrics before 3 months' do
+  describe 'Boundary: last three months from yesterday' do
+    it 'include metrics from 3 months ago starting in yesterday' do
       edition1 = create :edition, warehouse_item_id: 'warehouse_item_id1', date: 1.months.ago
-      create :metric, edition: edition1, date: (3.months.ago - 1.day)
+      create :metric, edition: edition1, date: (Date.yesterday - 3.months), upviews: 10
 
       recalculate_aggregations!
 
-      expect(subject.count).to eq(0)
+      expect(subject.sum(:upviews)).to eq(10)
+    end
+
+    it 'does not include metrics from 3 months ago - 2 days' do
```

Comment:
` 3 months and 2 days ago` is a little less confusing than `3 months ago - 2 days`